Eat_and_Greet/src/screens/ViewEventScreen/index.tsx

```diff
@@ -1,33 +1,88 @@
-import { Image, View, Text, TouchableOpacity } from 'react-native';
+import React, { useEffect, useState } from "react";
+import { Image, View, Text } from 'react-native';
+import { useRoute } from '@react-navigation/native';
 import { StatusBar } from 'expo-status-bar';
 import { Toolbar } from '../../comps/Toolbar/toolbar';
+import { db } from "../../firebase/config";
+import { getDoc, doc } from "firebase/firestore";
 import RoundedButton from '../../comps/RoundedButton/RoundedButton';
 import styles from '../../style';
 import localstyles from './style';
-import React from 'react';
+
+interface EventData {
```

Comment:
This is copied from Browse Event, wasn't sure if I could import a const and/or interface as import assumed they were functions